packages/core/src/contracts/ownedAddressTCRWithAppeals.ts

```diff
@@ -62,6 +63,12 @@ export class OwnedAddressTCRWithAppeals extends BaseWrapper<OwnedAddressTCRWithA
     return Voting.atUntrusted(this.web3Wrapper, await this.getVotingAddress());
   }
 
+  /**
+   * Returns Parameterizer instance associated with this TCR
+   */
+  public async getParameterizer(): Promise<Parameterizer> {
```

Comment:
Since the Paramizer's address is in the .json already - you can speed up this process by using the `await ParametizerContract.singleton()` call:
https://github.com/joincivil/Civil/blob/master/packages/core/src/templates/contract/partials/constructor.handlebars#L1